vue/components/ui/atoms/dropdown/dropdown.vue

```diff
@@ -381,8 +395,9 @@ export const Dropdown = {
             return {
                 separator: item.separator,
                 icon: Boolean(item.icon),
-                highlighted: this.highlightedData[index],
-                selected: this.selectedData[index]
+                highlighted: this.highlightedData[index] || item.highlighted,
+                selected: this.selectedData[index] || item.selected,
+                disabled: this.disabledData[index] || item.disabled

```

Comment:
Nice 👌 

Not related to this piece of code, but should the `onMouseenter` action highlight if the item is disabled?

Reply:
Think not, nice catch